src/routes/common/transactions.helper.spec.ts

```diff
@@ -1,15 +1,111 @@
+import { faker } from '@faker-js/faker';
 import * as axios from 'axios';
-import * as deployments from '@safe-global/safe-deployments/dist/libs';
+import * as deployments from '@safe-global/safe-deployments';
 import type { SingletonDeployment } from '@safe-global/safe-deployments/dist/types';
 
 import * as txHelpers from './transactions.helper';
 import * as safeHelpers from './safe.helper';
 import {
+  MOCK_CREATE_PROXY_WITH_NONCE_TX_CALL_DATA,
   MOCK_EXEC_TX_CALL_DATA,
   MOCK_MULTISEND_TX_CALL_DATA,
 } from '../../mocks/transaction-data.mock';
 
+jest.mock('@safe-global/safe-deployments', () => {
```

Comment:
My concern with this is that we are overly relying on mocks for the tests. We then have a tight coupling between the implementation and the tests.

My suggestion is to consider alternatives where we test the expected outcome and not the actual behavior. This can be considered outside of the scope of this PR but just something to keep in mind!

Reply:
You make a valid point. I have removed the spies.